gnoboard/src/hooks/use-gno.ts

```diff
@@ -8,6 +8,8 @@ import { QueryAccountResponse } from '@gno/api/gnomobiletypes_pb';
 import { DeleteAccountRequest, DeleteAccountResponse } from '@gno/api/gnomobiletypes_pb';
 import { QueryRequest } from '@gno/api/gnomobiletypes_pb';
 import { QueryResponse } from '@gno/api/gnomobiletypes_pb';
+import { RenderRequest } from '@gno/api/gnomobiletypes_pb';
```

Comment:
Can you clean a bit the imports by putting all in the same import?
```suggestion
import { ..., RenderRequest, QEvalRquest, ... } from '@gno/api/gnomobiletypes_pb';
```